flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/descriptors/ConnectTableDescriptor.java

```diff
@@ -136,14 +137,24 @@ public void createTemporaryTable(String path) {
 					" use registerTableSource/registerTableSink/registerTableSourceAndSink.");
```

Comment:
Hi @dawidwys , can we support temporary table without `schemaDescriptor`? we can derive schema from properties?

Reply:
That's what we are doing. The `schemaDescriptor` contain those properties.

Previously I think it was possible that the {{TableSource}} produced the {{TableSchema}} itself without getting any schema from properties. We cannot support this scenario anymore. Though I think all the connectors we provide so far require the schema properties anyway.